python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -98,73 +101,87 @@ def raw_emit_op(operator: JitOperator, f: TextIO, *, traits: List[str],
 
     You probably don't want to call this directly.
     """
-    emitter = TextEmitter(f)
-    p = lambda *args: emitter.print(*args)
-    op_name, td_def_name = operator.get_mlir_names()
+    p_td = lambda *args: emitter_td.print(*args)
+    op_name, cpp_class_name = operator.get_mlir_names()
 
     # Generate unique result names for ops with nameless results
     multiple_results = len(operator.returns) > 1
-    generic_result_name = lambda i: "result" + (str(i) if multiple_results else "")
-
-    p(f"def {td_def_name} : Torch_Op<{emitter.quote(op_name)}, [")
-    with emitter.indent():
-        with emitter.indent():
-            p(",\n".join(traits))
-        p("]> {")
-    with emitter.indent():
+
+    def generic_result_name(i): return "result" + \
```

Comment:
nit: I think the formatting here would be better if the `return` starts in the next line